### PR TITLE
Pal 710 ecr push

### PR DESCRIPTION
**Pull request by m78233**

### CODE REVIEW

File: clients-cat/pom.xml [hunk 1]
```diff
@@ -6,9 +6,6 @@
   ~ You may obtain a copy of the License at
   ~
   ~     http://www.apache.org/licenses/LICENSE-2.0
-  ~
-  ~ Unless required by applicable law or agreed to in writing, software
-  ~ distributed under the License is distributed on an "AS IS" BASIS,
```
Comment (pd104923) [inline clients-cat/pom.xml:11]: Is there a reason this has been removed from the license header?

**Review by pd104923 — CHANGES_REQUESTED** [review]

**Review by pd104923 — DISMISSED** [review]

File: Jenkinsfile [hunk 2]
```diff
@@ -82,20 +103,17 @@ timestamps {
...
                     if (sh(script: "git checkout ${GIT_BRANCH_NAME}", returnStatus: true) == 0) {
-                        container('docker-cmds') {
-                            configFileProvider([configFile(fileId: "${env.CONFIG_FILE}", variable: 'MAVEN_SETTINGS')]) {
-                                sh 'mvn -s $MAVEN_SETTINGS install -P quick'
-                            }
+                        READERS_REVISION = "BRANCH-${GIT_BRANCH_NAME_LOWER}-SNAPSHOT"
+                    } else {
+                        if (COMMON_REVISION == "BRANCH-${GIT_BRANCH_NAME_LOWER}-SNAPSHOT") {
+                            sh "mvn -s ${MAVEN_SETTINGS} -D revision=${READERS_REVISION} -D common.revision=${COMMON_REVISION} deploy"
+                            READERS_REVISION = "BRANCH-${GIT_BRANCH_NAME_LOWER}-SNAPSHOT"
```
Comment (pd104923) [inline Jenkinsfile:116]:
```suggestion
                    } else {
                        if (COMMON_REVISION == "BRANCH-${GIT_BRANCH_NAME_LOWER}-SNAPSHOT") {
                            sh "mvn -s ${MAVEN_SETTINGS} -D revision=${READERS_REVISION} -D common.revision=${COMMON_REVISION} deploy"
```
If the readers repo doesn't have a branch of the same name then why would the `READERS_REVISION` require the branch name to be in the snapshot? Wouldn't this just need to stay as `SNAPSHOT`? We've already done a deploy of readers using the `SNAPSHOT` revision value.

**Review by pd104923 — CHANGES_REQUESTED** [review]
1 question about the 2nd order dependency update.

**Review by pd104923 — DISMISSED** [review]
All good now

**Review by developer01189998819991197253 — DISMISSED** [review]

**Review by pd104923 — DISMISSED** [review]

**Review by developer01189998819991197253 — APPROVED** [review]

**Review by dev-958 — DISMISSED** [review]

**Pull request merged**